### ISSUE TITLE
Request infomation on the serial proposal

### ISSUE THREAD

**Issue by ayumihamasaki2019** [w3c/webappsec-feature-policy#342]
Hello,

I can't find any infomation about the serial proposal, although it's listed at the bottom of this web page: https://github.com/w3c/webappsec-feature-policy/blob/master/features.md

![image](https://user-images.githubusercontent.com/55155131/66241301-ab317800-e6f6-11e9-97de-451b8ac69394.png)

Does this proposal have a **github issue** or a **link** where I can find out more about this feature. It just seems to come out of thin air right now.

I'm trying to write up this spec for a technical article for my uni project.

Thanks.

**Comment by Malvoz** [comment]
Yes there doesn't seem to be any issue for discussion of this feature, here's the Chromium implementation entry though: https://bugs.chromium.org/p/chromium/issues/detail?id=884928#c21

**Comment by ayumihamasaki2019** [comment]
@Malvoz Thanks, that link answers all my questions. Hopefully someone could write a `serial.md` file for it at a later date if the proposal passes the Chrome teams findings.
